Dockerfile

```diff
@@ -26,7 +28,17 @@ COPY /nginx /etc/nginx/conf.d
 EXPOSE 8080
 EXPOSE 443
 
-COPY --from=builder /app/build /usr/share/nginx/html
-COPY --from=builder /app/env-config.js /usr/share/nginx/html/
+WORKDIR /usr/share/nginx/html
+
+# Copy built source
+COPY --from=builder /app/build .
+
+# Copy override ENV script with default values
+COPY ./env.sh .
+COPY .env .
```

Comment:
Perhaps the issue could be fixed by just this one line change, to copy `.env` file to Docker container

Reply:
this is how it currently works and though the root cause 
it's copied with default values like `localhos:4200` and when container starts it should be overridden by ENV variables from docker
if the script to form new `.env` file is not executed within the container it remains unchanged

Reply:
Can you clarify what do you mean?

Reply:
this is what I expect `_env_` variable to be on real env 
```
{
   PORT: "443"
   REACT_APP_API_URL: "https://visual-regression-tracker.com:4200"
   VRT_VERSION: "5.0.0-rc3"
}
```
despite the values that are copied into image initially https://github.com/Visual-Regression-Tracker/frontend/blob/master/.env

we set those values via docker-compose env variables https://github.com/Visual-Regression-Tracker/frontend/blob/master/docker-compose.yml#L13-L14

so when you start the app we have script executed to form new `_env_` https://github.com/Visual-Regression-Tracker/frontend/blob/master/package.json#L41

and we need the same inside the container

Reply:
Ok, I see. So the JavaScript file generation should happen later in the process, perhaps in the stage 2

Reply:
I went and tried #334

Reply:
@SwiLink I can see that `VRT_VERSION` is set 
what is the value in `.env` for `REACT_APP_API_URL`?
also `docker-compose` file should have it set as in template https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/blob/master/docker-compose.yml#L18

Reply:
Hey! Ok, here is my .env:
```
## Logging
# https://www.elastic.co/guide/en/elastic-stack/7.14/overview.html
ELK_VERSION=7.14.1
ELASTIC_URL=http://localhost:9200

## VRT version
VRT_UI_VERSION="5.0.0-rc3"
VRT_API_VERSION=5.0.0
VRT_MIGRATION_VERSION=5.0.0

## Frontend

# direct URL to backend with port (same as APP_PORT)
REACT_APP_API_URL=https://screenshots.wifox.org/api
# frontend port
PORT=8080

## Backend

# direct URL to frontend with port (same as PORT)
APP_FRONTEND_URL=https://screenshots.wifox.org
# backend port
APP_PORT=4200
# seed to generate JWT
JWT_SECRET=test
# user session lifetime
JWT_LIFE_TIME=1d
# max image size to upload
BODY_PARSER_JSON_LIMIT="5mb"

## DB

POSTGRES_PORT=5432
POSTGRES_USER=postgres
POSTGRES_PASSWORD=postgres
POSTGRES_DB=vrt_db
```

Reply:
Now it's working:)) Don't know what's the reason, but i know that later it will stop to work:)